src/connectors/archive.js

```diff
@@ -1,18 +1,34 @@
 'use strict';
 
+/**
+ * Example links to debug and test the connector:
+ *
+ * https://archive.org/details/AH013_sarin_sunday_-_the_lonely_hike
+ * Full album
+ *
+ * https://archive.org/details/AH003_corwin_trails_-_corwin_trails
+ * Full album with numeric prefixes
+ *
+ * https://archive.org/details/lp_everybody-knows-this-is-nowhere_neil-young-crazy-horse-robin-lane
+ * Full album with artist suffixed in track names
+ *
+ * https://archive.org/details/dont-lie-beets-produce-ny-mix
+ * Single track
+ */
+
 const artistSelectors = [
 	'.key-val-big a span',
```

Comment:
@inverse What this and `.jwrowV2.playing .ttl` selectors are for? I haven't find anything where they could be used.

Reply:
Looks like it's changed to 

```
const tracksSelector = '.jwlistV2 .ttl';
```

Reply:
I can see it on https://archive.org/details/AH003_corwin_trails_-_corwin_trails/Oasis_is_Myth.mp3

It looks like that's broken though due to the extra whitespace. 

We could either update line 97 to `if (!numericTrackRegex.test(track.textContent.trim())) {` or relax the regex matching.

Reply:
Yes, I noticed two types of players on archive.org, where `.jwrowV2.playing .ttl` and `.audio-track-list .selected .track-title` are used as *track title* selectors. But I still can't see anything that match the `.key-val-big a span` selector.

Reply:
> It looks like that's broken though due to the extra whitespace.

Interesting, it was working fine when I tested changes. :)

Reply:
I wonder why it's behaving differently...